app/components/playerWelcome.tsx

```diff
@@ -7,12 +8,10 @@ export type Player = { id?: number; name: string; surname: string };
 
 export function PlayerWelcome({ players }: PlayersProps) {
   return (
-    <div>
-      <div>
+      <HelloDiv>
```

Comment:
W nazwie Stylowanego komponentu już nie dodawałabym nazw elementów html-owych, tak jak "div", "p" itd. Chyba, że nazwa elementu html'owego daje nam info o tym elemencie np, "ManagerButton" oprócz tego, że rzeczywiście jest to button, to daje nam info o tym, że jest to element, który coś robi, np. można na niego kliknąć itp.